impeller/aiks/paint.cc

```diff
@@ -121,6 +124,32 @@ std::shared_ptr<Contents> Paint::WithColorFilter(
                                               absorb_opacity);
 }
 
+std::shared_ptr<FilterContents> Paint::MaskBlurDescriptor::CreateMaskBlur(
+    std::shared_ptr<TextureContents> texture_contents,
+    const std::shared_ptr<ColorFilter>& color_filter) const {
+  Scalar expand_amount = GaussianBlurFilterContents::CalculateBlurRadius(
+      GaussianBlurFilterContents::ScaleSigma(sigma.sigma));
+  texture_contents->SetSourceRect(
```

Comment:
Do we want to copy this contents instead of mutating it?

Reply:
These objects are created on the fly for aiks rendering, so in theory that should cause no problems? Entity objects aren't ever exposed in the public API, its not possible to cache and reuse them